build.gradle

```diff
@@ -0,0 +1,70 @@
+/**
```

Comment:
Please add a license/copyright header (you can copy from one of the Java files) with attribution to those who were involved in developing this file.

Reply:
Finished